bin/azurite/wait

```diff
@@ -6,29 +6,29 @@ pushd $(git rev-parse --show-toplevel) > /dev/null
 
 set -e
 
-azurite_endpoint=http://localhost:10000
+azurite_endpoint="http://localhost:10000"
```

Comment:
just formatting changes in this file, i didn't do the whole thing but can if desired. it's late.